guacamole-common-js/src/main/webapp/modules/Keyboard.js

```diff
@@ -625,6 +653,12 @@ Guacamole.Keyboard = function(element) {
     }
 
     function keysym_from_keycode(keyCode, location) {
+
+        // Map Clear on Mac to NumLock
+        if (keyCode === 12 && is_mac) {
```

Comment:
Sure, but if we universally map `Clear` to `Num Lock`, that will encumber its use as the key it actually *is* when connecting to another Mac. It would make sense to do this if its existing mapping were wrong, but if the key is properly detected, then this is working correctly as is.

Perhaps this sort of remapping should instead be handled within the RDP keymap, where we can be reasonably certain that there is no `Clear` and, even if there is, it's handled as a scancode and will work anyway?

We wouldn't want to break use of VNC into Macs, nor the behavior of `Guacamole.Keyboard` in general (which should only be concerned with achieving as close to 100% accurate keyboard event recognition as possible).